From eac40bb3773af39d290bd832c86fb016aa367ca4 Mon Sep 17 00:00:00 2001
From: Richard Lowe <richlowe@richlowe.net>
Date: Thu, 25 Apr 2024 19:00:13 +0000
Subject: [PATCH] 16498 closed nfs dmod/kmod are still delivered Reviewed by:
 Andy Fiddaman <illumos@fiddaman.net> Reviewed by: Bill Sommerfeld
 <sommerfeld@hamachi.org> Approved by: Robert Mustacchi <rm@fingolfin.org>

---
 exception_lists/closed-bins | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/exception_lists/closed-bins b/exception_lists/closed-bins
index f24001116d..bdc18de137 100644
--- a/exception_lists/closed-bins
+++ b/exception_lists/closed-bins
@@ -42,6 +42,7 @@
 ./kernel/drv/usbser_edge
 ./kernel/kmdb/mpt
 ./kernel/kmdb/nfs
+./kernel/kmdb/amd64/nfs
 ./kernel/misc/klmmod
 ./kernel/misc/amd64/klmmod
 ./kernel/misc/klmops
@@ -133,6 +134,7 @@
 ./usr/lib/localedef/src/iso_8859_1/localedef.src
 ./usr/lib/mdb/kvm/mpt.so
 ./usr/lib/mdb/kvm/nfs.so
+./usr/lib/mdb/kvm/amd64/nfs.so
 ./usr/lib/nfs/lockd
 ./usr/lib/snmp
 ./usr/lib/snmp/mibiisa
-- 
2.46.0

